examples/gno.land/r/sys/users/admin.gno

```diff
@@ -0,0 +1,101 @@
+package users
+
+import (
+	"std"
+
+	"gno.land/p/demo/ownable"
+	p "gno.land/p/demo/pausable"
+)
+
+const (
+	adminAddr              = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq" // @moul
+	WhitelistAddEvent      = "WhitelistAdd"
+	WhitelistDeleteEvent   = "WhitelistDelete"
+	VerifyFuncUpdatedEvent = "VerifyFuncUpdated"
+
+	gnolandUsers = "gno.land/r/gnoland/users" // preregistered with store write perms
+)
+
+var (
+	admin    = ownable.NewWithAddress(adminAddr) // Package owner
```

Comment:
you can keep it until we define an idiomatic way to make a contract managed by a dao, but add a comment